### ISSUE TITLE
Specifying a Launch Screen Image for safari

### ISSUE THREAD

**Issue by san650** [san650/ember-web-app#36]
Add a configuration option to add a startup image

```html
<link rel="apple-touch-startup-image" href="/launch.png">
```

Specifications https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html#//apple_ref/doc/uid/TP40002051-CH3-SW6

**Comment by sdebarros** [comment]
It seems this tag is not working on the latest versions of iOS 
https://gist.github.com/tfausak/2222823#gistcomment-1582419
https://forums.developer.apple.com/thread/23924

😞

**Comment by san650** [comment]
@sdebarros I'll look for alternatives...

Maybe having this option for older iOS devices is still a good idea though, what do you think?

**Comment by sdebarros** [comment]
@san650 agree, though it seems to be failing from iOS 9 onward, not sure how many devices have iOS 8 or less

**Comment by st-h** [comment]
From what I have read the last days, specifying `apple-touch-startup-image` was broken for several iOS releases. I just checked and can confirm that manually adding `<link rel="apple-touch-startup-image" ... >` to `index.html` allows to specify startup splash screens on iOS `11.4.1`. So this feature would still be very useful, or does anyone know of any other way to specify a startup splash screen image? Specifying them as icon does not work.
